lineman/app/components/group_page/group_page.scss

```diff
@@ -59,12 +59,23 @@
   @include card;
 }
 
+.group-page__threads {
+  padding: 10px 0 17px;
+  h2 {
```

Comment:
Is it the same heading style as used elsewhere in the app? If so we have .lmo-card-heading